app/services/school_groups/recent_usage_csv_generator.rb

```diff
@@ -0,0 +1,76 @@
+module SchoolGroups
+  class RecentUsageCsvGenerator
+    def initialize(school_group:, metric: 'change')
+      raise unless %w[change usage cost co2].include?(metric)
+      @school_group = school_group
+      @metric = metric
+    end
+
+    def export
+      CSV.generate(headers: true) do |csv|
+        csv << headers
+        @school_group.schools.visible.order(:name).each do |school|
+          recent_usage = school&.recent_usage
+
+          row = []
+          row << school.name
+          if fuel_types.include?(:electricity)
+            if recent_usage&.electricity&.week&.has_data
+              row << Nokogiri::HTML(recent_usage&.electricity&.week.send(@metric)).text
```

Comment:
As this is to undo the formatting done by SummaryTableData when presenting the data for display, I suggest we make some changes to that class so we can return the unformatted numbers without %.

E.g. this line in that class formats the raw number from the database:

`change: format_number(fetch(fuel_type, period, :percent_change), :comparison_percent, :text),`

We could add:

`change_numeric: FormatEnergyUnit.format(:comparison_percent, VALUE_HERE, :text, false, true, :target)`

Which formats the value for text, e.g. "100%").

Then in the CSV export we can use `change_numeric` rather than `change`?